cmd/cephcsi.go

```diff
@@ -131,6 +131,7 @@ func init() {
 	flag.BoolVar(&conf.Version, "version", false, "Print cephcsi version information")
 	flag.BoolVar(&conf.EnableProfiling, "enableprofiling", false, "enable go profiling")
 
+	flag.BoolVar(&conf.SupportNsenter, "supportnsenter", false, "Use nsenter to map and mount -t operation")
```

Comment:
Instead of a `bool`, I would prefer to have an option like `--network-pid=${pid}` which then translates to a namespace-path like `/proc/${pid}/ns/net`. This should make it easy to pass the parameter as an (environment) variable, and no need to add a new option in the configuration file/ConfigMap.

Reply:
This is just to say we support nsenter or not and the actual configuration will be fetched from the CSI configmap which has the clusterID and mon mapping. This helps to support multiple ceph clusters in different multus network